lib/std/special/c.zig

```diff
@@ -528,7 +527,51 @@ fn clone() callconv(.Naked) void {
                 \\  blr
             );
         },
+        .sparcv9 => {
+            // Implementation by @LemonBoy (https://github.com/LemonBoy)
+            // __clone(func, stack, flags, arg, ptid, tls, ctid)
+            //           i0,    i1,    i2,  i3,   i4,  i5,   sp
 
```

Comment:
misc empty line?

Reply:
I'm following the formatting in the aarch64 and riscv64 part. Should I remove it?

Reply:
Nevermind, removed it.

Reply:
Nevermind, removed it.